Firestore/core/src/firebase/firestore/util/filesystem_posix.cc

```diff
@@ -95,6 +95,17 @@ Path TempDir() {
 }
 #endif  // !defined(__APPLE__)
 
+Status GetFileSize(const Path& path, off_t* size) {
+  struct stat st;
```

Comment:
Initialize this so automated tools don't ding us for possibly reading uninitialized memory.

```c++
struct stat st{};
```

Reply:
Done.